plugins/dapper/README.md

```diff
@@ -0,0 +1,168 @@
+# Dapper Plugin for SBOM Surfactant
+
+A plugin for Surfactant that uses [Dapper](https://github.com/LLNL/dapper) to identify software packages from binary files by querying comprehensive package databases.
+
+## Overview
+
+The Dapper plugin enhances Surfactant's SBOM generation by identifying which software packages provide specific binary files. It queries multiple Linux distribution databases (Ubuntu, Debian) to find package information for ELF files, with planned support for Windows PE files via NuGet databases.
+
+## Features
+
+- **Multi-distribution package detection**: Queries all available Linux distributions (Ubuntu Focal/Jammy/Noble, Debian Buster/Bullseye/Bookworm)
+- **Comprehensive results**: Returns all matching packages across distributions
+- **File normalization**: Handles various file naming conventions through intelligent normalization
+- **Structured output**: Provides detailed package information including package names, versions, and source distributions
+
+## Prerequisites
+
+### Install Dapper and Datasets
+
+Before using this plugin, you need to install Dapper and download the package databases:
+
+1. Install dapper-python (required for database queries):
+   ```bash
+   pip install dapper-python
+   ```
+
+2. Clone and build Dapper:
+   ```bash
+   git clone https://github.com/LLNL/dapper.git
+   cd dapper
+   cargo build --release
+   ```
+
+3. Download required datasets (must be run from the dapper directory):
+   ```bash
+   # List available datasets
+   cargo run -- . --list-available-datasets
+
+   # Install specific Linux datasets
+   cargo run -- . --install ubuntu-focal
+   cargo run -- . --install ubuntu-jammy
+   cargo run -- . --install ubuntu-noble
+   cargo run -- . --install debian-bookworm
+
+   # Install all available datasets
+   cargo run -- . --install all
+   ```
+
+## Installation
+
+In the same virtual environment that Surfactant was installed in, install this plugin:
+
+```bash
+# From PyPI (when available)
+pip install surfactantplugin-dapper
+
+# From GitHub
+pip install git+https://github.com/LLNL/Surfactant#subdirectory=plugins/dapper
+
+# For developers making changes to this plugin
+git clone https://github.com/LLNL/Surfactant.git
+cd Surfactant/plugins/dapper
+pip install -e .
+```
+
+
+## Output Format
+
+The plugin adds package information to the metadata field of software entries in the SBOM. For each binary file, it provides:
+
+```json
+{
+  "dapper_packages": [
+    {
+      "package_name": "coreutils",
+      "full_package_name": "coreutils/focal 8.30-3ubuntu2",
```

Comment:
I think we should remove the full_package_name, the version information is misleading (we don't have any indication about what the version number is or should be).